changelog/unreleased/features/2133--status-grand-total.md

```diff
@@ -0,0 +1,2 @@
+The output of `vast status --detailed` now displays the total number of events
```

Comment:
I think that information is important enough to be displayed on the `info` level.